cmd/curio/run.go

```diff
@@ -67,6 +67,12 @@ var runCmd = &cli.Command{
 			EnvVars:     []string{"CURIO_NODE_NAME"},
 			DefaultText: translations.T(""),
 		},
+		&cli.StringFlag{
+			Name:    "csp",
+			Usage:   translations.T("Content Security Policy setting"),
+			Value:   "inline",
+			EnvVars: []string{"CURIO_CSP"},
+		},
```

Comment:
does this need a "Content Security Policy setting" entry in the translations somewhere? I don't know how that works here but you should outline the possible values and the implications, particularly just how YOLO "off" is going to be